page creation for weather data

TylorJB opened the pull request.

Rmhibbert reviewed (changes requested):
The about page needs to be removed from the home page to avoid any clashes. Please see below screenshot
<img width="1683" height="545" alt="image" src="https://github.com/user-attachments/assets/f2386e93-ee5f-4ed3-bd51-39a35eaf9f99" />

Future improvements would be to add additional number to the side of the graphs for more details and to add more context to data readings for example the gas percentage, percentage of what?? These are out of scope

<img width="155" height="312" alt="image" src="https://github.com/user-attachments/assets/a0414ad7-e258-42dc-9ed0-9c44f70904fc" />

<img width="454" height="99" alt="image" src="https://github.com/user-attachments/assets/7ab2b556-6aa8-478d-a686-be23b31b32b3" />

TylorJB commented:
I have made the changed you specified. I removed the hard link to the about page as we ready for merge. 
<img width="1401" height="490" alt="{0031E6B7-9828-4BE8-AD6D-A3527E979965}" src="https://github.com/user-attachments/assets/7250f748-77d0-4b1e-b36b-7c9ecc3188e8" />

I changed the gas unit of measurement to ppm. 
<img width="453" height="171" alt="{A6C58352-1029-4CC1-9952-03EA26FE5CF6}" src="https://github.com/user-attachments/assets/8f3c86dc-24c8-4942-a495-f9ee83840467" />

We will fix the other changes next sprint as you are correct in it being out of scope. Hopefully this is ready for merge. Let me know if its good to go.

Rmhibbert reviewed (approved):
I am happy to approve this request as I can see the changes have been made and the issue no longer exists
